docs/dependencies.md

```diff
@@ -1,80 +1,196 @@
-## 1. Download and Install Dependencies
+# Particle toolchain dependencies
 
-Building the firmware locally requires these dependencies ot be installed:
+There are two options for building the Particle firmware.
 
-1. [GCC for ARM Cortex processors](#1-gcc-for-arm-cortex-processors)
-2. [Make](#2-make)
-3. [Device Firmware Upgrade Utilities](#3-device-firmware-upgrade-utilities)
-4. [Zatig](#4-zatig) (for Windows users only)
-5. [Git](#5-git)
-6. [Command line tools](#6-command-line-tools)
+1. You can download and install the dependencies natively
+1. You can install Docker and run the toolchain in a container _(with all dependencies supplied therein)_
```

Comment:
2?

Reply:
In markdown you just use the number 1 over and over again, and it will render it correctly. This allows you to rearrange steps without having to renumber the whole list.

Reply:
While I agree that it renders correctly, markdown is still a human-readable format and I don't see any reason why the enumeration shouldn't be correct even in plain-text.